addons/udes_stock_cron/models/stock_picking.py

```diff
@@ -30,54 +30,55 @@ def _find_unreservable_moves(self, batch_size=1000, return_fast=False):
 
         # Create empty record sets for moves.
         unreservable_moves = Move.browse()
+        for picking_type, pickings in self.groupby(lambda p: p.picking_type_id):
+
+            location = picking_type.default_location_src_id
+            stock_for_products = defaultdict(int)
```

Comment:
Should this go outside? If two picking types have the same source location, we could potentially count the same stock as available twice.